packages/salesforcedx-vscode-soql/src/editor/queryRunner.ts

```diff
@@ -11,10 +11,7 @@ import { QueryResult } from 'jsforce';
 import * as vscode from 'vscode';
 
 export class QueryRunner {
-  constructor(
-    private connection: Connection,
-    private document?: vscode.TextDocument
```

Comment:
This was left over from previous PR when I thought the Query runner would also save files.